DigitalLearningSolutions.Web.AutomatedUiTests/AccessibilityTests/EditCourseAdminFieldsAccessibilityTests.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace DigitalLearningSolutions.Web.AutomatedUiTests.AccessibilityTests
+{
+    using DigitalLearningSolutions.Web.AutomatedUiTests.TestHelpers;
+    using FluentAssertions;
+    using Selenium.Axe;
+    using Xunit;
+
+    public class EditCourseAdminFieldsAccessibilityTests : AccessibilityTestsBase
+    {
+        public EditCourseAdminFieldsAccessibilityTests(SeleniumServerFactory<Startup> factory) : base(factory) { }
+
+        [Fact]
+        public void EditAdminField_journey_has_no_accessibility_errors()
+        {
+            // Given
+            Driver.LogUserInAsAdminAndDelegate(BaseUrl);
+            const string startUrl = "/TrackingSystem/CourseSetup/100/AdminFields/1/Edit";
+
+            // When
+            Driver.Navigate().GoToUrl(BaseUrl + startUrl);
+            ValidatePageHeading("Edit course admin field");
+            var editPageResult = new AxeBuilder(Driver).Analyze();
+
+            Driver.ClickButtonByText("Bulk edit");
+            ValidatePageHeading("Configure answers in bulk");
+            var bulkAdditionResult = new AxeBuilder(Driver).Analyze();
+
+            editPageResult.Violations.Should().BeEmpty();
```

Comment:
Need a variation of the lines you deleted in the latest commit added back in here, checking the page title matches the Edit page title. (I presume this was removed to make the test pass since it was using the registration prompts title).

The purpose of the extra chunk was to verify that the submit button on the Bulk page redirected you back to the correct place, we've lost that check for the AdminFields journey by removing it. (There is an assertion in the ValidatePageHeading method).

Reply:
@AlexJacksonDS Ah, I was confused about that chunk because the wording of my button on the bulk page is different, added it back in and it is passing.

Also, earlier I changed one field in the Customisations table in the remote test database (adding a CourseField1PromptID value to the customisation with ID 100), so the edit course admin field page would exist for the test to check. Is this okay and do I have to document this change somewhere?

Reply:
The preferred way to do this would be for the tests to access a prompt with existing data that works for them. (I don't know if this exists in this case, but it is worth checking).

Otherwise, I think the process is as below, but there may be something extra I've missed:
- Add a script to the SQLScripts folder that will do whatever changes are needed
- Update the test database backup we use in the Zero to Hero
- Let everyone know this exists in slack
- (I assume tell Kevin at some point?)